physionet-django/physionet/test_context_processor.py

```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase
```

Comment:
The "static page" changes now seem unrelated to this pull request. Could these be moved to a new pull request?